ReadsUtils.spec

```diff
@@ -69,6 +69,13 @@ module ReadsUtils {
         - OR -
         fwd_file - a local path to the reads data file: either single end
             reads, forward/left reads, or interleaved reads.
+        - OR - 
+        fwd_file_url - a download link that contains reads data file:
+            either single end reads, forward/left reads, or interleaved reads.
+        download_type - download type ['Direct Download', 'FTP', 'DropBox', 'Google Drive']
+        - OR - 
+        fwd_staging_file_name - reads data file name in staging area:
+            either single end reads, forward/left reads, or interleaved reads.
```

Comment:
Clarify the text at the beginning of the documentation here to cover the new options - only covers the previous options currently.